src/pages/sighting/AssetGroupSighting.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useParams, useHistory } from 'react-router-dom';
+
+import SightingCore from './SightingCore';
+import useAssetGroupSighting from '../../models/assetGroup/useAssetGroupSighting';
+
+export default function AssetGroupSighting() {
+  const history = useHistory();
+  const { id } = useParams();
+
+  const {
+    data: assetGroupSightingData,
+    loading: assetGroupLoading,
+    error: assetGroupError,
+    statusCode: assetGroupStatusCode,
+  } = useAssetGroupSighting(id);
+
+  const sightingId = assetGroupSightingData?.sightingGuid;
+
+  useEffect(
```

Comment:
automatically redirect to sightings page if AGS has already been turned into a sighting